templates/app/webpack-default/template/src/profile.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * This example demonstrates how a simple profile view can be created with
+ * Titanium. The profile includes
+ *
+ * - Profile header
+ * - A styled TableView with items that open new windows
+ * - ScrollView to allow scrolling of the whole view on smaller devices
+ */
+
+/* global tabGroup */
+
+import { createWindow, createTab, faIcon, runs } from './utils';
+
+const title = 'Profile';
+const window = createWindow({ title, layout: 'vertical' });
+
+const scrollView = Ti.UI.createScrollView({ layout: 'vertical' });
+window.add(scrollView);
+
+// header profile pic & username
+const profilePic = Ti.UI.createImageView({
+	image: 'images/profile.jpg',
+	top: 30,
+	width: 120,
+	height: 120,
+	borderRadius: 60
+});
+scrollView.add(profilePic);
+const username = Ti.UI.createLabel({
+	text: 'Verona Blair',
+	font: {
+		fontSize: 28
```

Comment:
Another nitpick unrelated to this PR: why do we name the property `fontSize` if it's going to be inside a `font` object. Seems redundant.